guides/development-workflow.md

```diff
@@ -118,25 +120,24 @@ general guidelines.
    git push origin <branch-name>
    ```
 
-1. create a [pull request](#pull-requests) into the `develop` branch
+1. create a [pull request](#pull-requests) into the active development branch
 1. link the Trello card to the new pull request, linking directly or by using
    the Trello
    [power up](https://help.trello.com/article/1021-integrating-trello-with-zendesk)
-1. move the Trello card into the 'Awaiting review column'
+1. update the card to indicate it's now ready for review by a peer
 
 ### Reviewing another developer's pull request
 
-- move the Trello card into the 'Under code review' column
-- find and read the Trello card's acceptance criteria
+- update the Trello card to indicate you're now looking at it
+- find and read the Trello card's user need or acceptance criteria
 - find and read through the pull request, carrying out a
   [code review](#code-review)
-  - if it is ready:
-    - press the merge button
-    - delete the leftover branch
-    - move the Trello card into 'Deployed to Staging'
-  - if it's not ready:
-    - leave feedback
-    - mention the author so they're aware
+  - if you think it's ready:
+    - leave your feedback (positive comments are great to receive)
+    - approve the pull request
+  - if you think it's not ready:
+    - leave your feedback
+    - consider letting the author know by Slack that it's back with them
```

Comment:
Oh yes, for sure. When you should use approve vs comment vs request changes would be great to have guidance on.

For me:

- Approve: I'm happy for this to be merged, without any of the things I commented on being addressed (or, being merged by the author after they've addresses those things, if that's the way they want to do it)

Reply:
Interesting. I use "comment" to mean "I have suggestions, but it's your call", whereas I'd use a "request changes" if I'm saying "this is actively dangerous, or is going to cause issues if it's merged and would like my comments addressed first, please".

Reply:
So we agree on "request changes". When would you use "approve"? Only when you have no suggestions?

Reply:
I no longer ever use "request changes" but prefer to use "approve" or "comment" and make myself very clear with my wording what I expect. I percieve the red cross and demanding tone as needlessly aggressive and not a nice experience.

Reply:
- Approve: I'm happy for it to go, I may have optional suggestions or even low impact suggestons I trust you to action and the prior approval is then a convient mechasim by which no second review is required
- Comment: If I'm passing and don't feel confident enough to make a green decision or if there's any reason I think we must take action and it's important that is rechecked.
- Request changes: never

Reply:
I'd use "approve" when I have no strong suggestions or remarks. It's a scale of comfort for me, I think.
- Approve: I'd have no objections to this being merged as is, but here are maybe some extra things to think about
- Comment: I have thoughts on this thing that I'd probably be a bit grumpy if you just ignored, but my feedback isn't showstopping if someone else thinks it's good to go
- Request changes: I would consider it an issue if this got merged because it's actively dangerous (I basically never do this level of review)

Reply:
Worth pointing out that I don't think GitLab has "Request changes" but instead looks to only have comment or approve functions.

Given we have subtle differences in our application of these tools and I presume the rest of the team will be similar, I don't think it will benefit us to be too prescriptive here. That said I do think we should reference _a_ way to interpret and use these tools for those looking for guidance. I will add that with the suggestion that if it becomes a hot potatoe it should probably be extracted into a new pull request.

Reply:
I've tried to summarise where I think we got to:

* Approve - I am confident that this pull request can safely be merged in. I MAY also leave optional suggestions which the reviewer MAY want to consider
* Comment - I may not be confident in the change, in which case I should explain why this SHOULD NOT be merged yet and what SHOULD be considered. Alternatively I might not feel confident enough to approve this pull request but MAY have suggestions that SHOULD be considered
* Request changes (not available on GitLab) - This may be used to further indicate that this pull request SHOULD NOT be merged. It may contain severe issues such as security concerns or bugs which might have a serious impact. This option is used to explicitly nullify any other review that chose approval